service/device/registry/test/src/test/java/org/eclipse/kapua/service/device/registry/test/CucumberWithPropertiesForDeviceRegistry.java

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2019 Eurotech and/or its affiliates and others
+ * Copyright (c) 2020 Eurotech and/or its affiliates and others
```

Comment:
I see this only now - please change header to " * Copyright (c) 2019, 2020 Eurotech and/or its affiliates and others"